apps/api/src/applicant/ienapplicant.service.ts

```diff
@@ -618,7 +618,7 @@ export class IENApplicantService {
     }
 
     nursing_educations.push({
-      name: 'Diploma/Certificate of Nursing',
+      name: 'Education',
```

Comment:
Does this mean the case that they don't know the title of education?

Reply:
That is correct.